### PR TITLE
add PHP7.2 to tests

### PR DESCRIPTION
**Pull request by davitol**
add PHP7.2 to tests

### CODE REVIEW

**Comment by karakayasemi** [comment]
I fixed some issues on Travis in this pull request: #53 . We can rebase this after the #53 is merged. I believe, tests will pass after rebase.

File: .travis.yml [hunk 1]
```diff
@@ -49,9 +49,13 @@ script:
...
   include:
-    - php: 7
-      env: "DB=pgsql PHP_COVERAGE=TRUE"
-    - php: 7
-      env: "DB=mysql"
+    - php: 7.1
+      env: DB=sqlite CORE_BRANCH=master
+    - php: 7.2
+      env: DB=sqlite CORE_BRANCH=master
+    - php: 7.1
```
Comment (karakayasemi) [inline .travis.yml]: stable10 php7.1-sqlite and php7.2-sqlite are already in the test environments. These two matrix entries are redundant. See https://github.com/owncloud/twofactor_totp/blob/01edf7bb19717ec30c5b15951d88dd893e5ff400/.travis.yml#L7 . 
But there is no test for postgresql, mysql and oracle. IMHO, we should add these databases to the test matrix.

File: appinfo/info.xml [hunk 2]
```diff
@@ -24,7 +24,7 @@ This extension provides the most simple way to add a second factor to user login
 	</settings>
 
 	<dependencies>
-		<owncloud min-version="10.0" max-version="10.0" />
+		<owncloud min-version="10.0" max-version="10.1" />
 		<php min-version="5.4" max-version="7.2" />
```
Comment (patrickjahns) [inline appinfo/info.xml]: Is min version of php 5.4 still valid?

**Comment by individual-it** [comment]
rebased on master

File: appinfo/info.xml (hunk 2 above)
Comment (individual-it) [reply appinfo/info.xml]: I think we should have the min-version of PHP set to what we are testing, I've changed it to 5.6

**Comment by individual-it** [comment]
fixed the matrix, set php-min-version to 5.6 and squashed the commits
@patrickjahns @karakayasemi please review again

**Pull request merged**